src/components/v2/BscLink/index.tsx

```diff
@@ -0,0 +1,34 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import Typography from '@mui/material/Typography';
+
+import { isOnTestnet } from 'config';
+import { Icon } from '../Icon';
+import { useStyles } from './styles';
+
+export interface IBscLinkProps {
+  hash: string;
+  className?: string;
+}
+
+export const BscLink: React.FC<IBscLinkProps> = ({ hash, className }) => {
+  const styles = useStyles();
+
+  const bscScanUrl = `https://${isOnTestnet ? 'testnet.' : ''}bscscan.com/address/${hash}`;
```

Comment:
can be a good util that wraps urls. wouldn't it used in other places?

Reply:
Yeah good point. It could be reused in the footer already. I'll make the change